setup.cfg

```diff
@@ -0,0 +1,27 @@
+[metadata]
+name = herbie
+version = 0.1
+description = Herbie is a central data store for business data in a distributed service architecture.
+long_description = file: README.rst
+url = https://github.com/herbie
+author =
+author_email =
+license = MIT
+
+[options]
+python_requires = >=3.6
+include_package_data = true
+packages = find:
+install_requires =
+    django >= 3.0
+    django-environ >= 0.4
+    django-model-utils >= 4.0
+    djangorestframework >= 3.11
+    jsonschema >= 3.2
+    psycopg2 >= 2.8
```

Comment:
will this dependency definition replace poetry / requirements.txt? do we need an extra requirements.txt for these dependencies. And is this the reason for removing poetry?

Reply:
Yeah, you are right. With my refactoring, herbie will be a package that needs to be integrated in a Django project. The setup.cfg defines the dependencies of the package.